spec/rubocop/cop/rspec/capybara/specific_finders_spec.rb

```diff
@@ -66,14 +147,32 @@
       ^^^^^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
       find('[visible][id=some-id]')
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
-      find('[id=some-id][class=some-cls][focused]')
-      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `find_by` over `find`.
+      find('[id=some-id][visible][focused]')
```

Comment:
CSS selector `[visible]` means the element has an attribute named `visible` (e.g. `<span visible="yes">`), NOT "the element is actually visible". CSS does not have `:visible` pseudo-class ---  it's a jQuery's extension.
And also, `[focused]` means the element has an attribute named `focused`, NOT "the element is now focused". `:focus` pseudo-class should be used instead.

Reply:
That is indeed true. I guess I was mistaken. Thanks for pointing that out to me.